docs/tools/tutorials/adding-terms.md

```diff
@@ -131,15 +131,15 @@ else:
 ```
 
 We're using the `MetdataChangeProposalWrapper` to change entities in this example.
-For more information about the `MetadataChangeProposal`, please refer to [MetadataChangeProposal & MetadataChangeLog Events](/docs/advanced/mcp-mcl/)
+For more information about the `MetadataChangeProposal`, please refer to [MetadataChangeProposal & MetadataChangeLog Events](https://datahubproject.io/docs/advanced/mcp-mcl/)
 
 
 
 ## Add Terms With GraphQL
 
 :::note
 Please note that there are two available endpoints (`:8000`, `:9002`) to access GraphQL.
-For more information about the differences between these endpoints, please refer to [DataHub Metadata Service](/docs/metadata-service/#graphql-api)
+For more information about the differences between these endpoints, please refer to [DataHub Metadata Service](https://datahubproject.io/docs/metadata-service/#graphql-api)
```

Comment:
hey! The problem here is that you didn't add .md at the end of the file- if you do that you can add the # to deeplink like you would for a normal link- see here: 
<img width="487" alt="image" src="https://user-images.githubusercontent.com/101486603/224199168-5b0e0c32-298a-480e-aa1c-637402519308.png">